common/src/main/java/io/netty/util/concurrent/GlobalEventExecutor.java

```diff
@@ -34,7 +35,7 @@
  * task pending in the task queue for 1 second.  Please note it is not scalable to schedule large number of tasks to
  * this executor; use a dedicated executor.
  */
-public final class GlobalEventExecutor extends AbstractScheduledEventExecutor {
+public final class GlobalEventExecutor extends AbstractScheduledEventExecutor implements OrderedEventExecutor {
```

Comment:
Nice idea, @njhill

Reply:
What if a blocking IO task is submitted, won't it jam up the executor now?

Reply:
It should not be any different then at the moment... why you think there is an issue now ?

Reply:
I think it depends on the underlying executor, but the fact it now implements OrderedEventExecutor doesn't it mean it cannot run submitted tasks OOO which may result in tasks queuing up by implementing this interface?

Reply:
@johnou it's just a marker interface which doesn't change any behaviour itself. `GlobalEventExecutor` _already_ behaves in the manner it specifies (serial, in-order execution) and hence might as well implement it.

Reply:
let me actually do this in another PR as it is not really related.

Reply:
https://github.com/netty/netty/pull/8952